### PR TITLE
XW 4591 | Removing jQuery occurrences from Ember components

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: app/components/article-media-gallery.js [hunk 1]
```diff
@@ -97,7 +96,7 @@ export default Component.extend(
 				// Make sure that some math error above doesn't cause images to not load
 				this.set('numberOfItemsRendered', totalNumberOfItems);
 			} else {
-				$this.off('scroll');
+				this.element.removeEventListener('scroll');
```
Comment (hakubo) [inline app/components/article-media-gallery.js]: I think function ref has to be passed as a 2nd param

File: app/components/article-media-gallery.js [hunk 2]
```diff
@@ -37,9 +37,7 @@ export default Component.extend(
 		 * @returns {void}
 		 */
 		didRender() {
-			this.$().on('scroll', () => {
-				debounce(this, 'onScroll', 100);
-			});
+			this.element.addEventListener('scroll', this.debouncedScroll);
```
Comment (latata) [inline app/components/article-media-gallery.js]: We'll lose `this` here. You should bind `debouncedScroll` to `this` (`this.debouncedScroll  = this. debouncedScroll.bind(this)`)

File: app/components/category-members-grouped.js [hunk 3]
```diff
@@ -30,11 +30,15 @@ export default Component.extend(
...
 					.then(() => {
-						const navHeight = $('.site-head').outerHeight() + $('.site-head-fandom-bar').outerHeight(),
-							scrollTop = this.$().offset().top - navHeight;
+						const navHeight = document.querySelector('.site-head').offsetHeight +
+							document.querySelector('.site-head-fandom-bar').offsetHeight,
+							scrollTop = offset(this.element).top - navHeight;
 
 						run.scheduleOnce('afterRender', this, () => {
-							$('html, body').animate({scrollTop});
+							window.scroll({
```
Comment (latata) [inline app/components/category-members-grouped.js:38]: don't we need polyfill for that? http://iamdustan.com/smoothscroll/

File: app/components/featured-content.js [hunk 4]
```diff
@@ -116,10 +116,10 @@ export default Component.extend(
 		},
 
 		updatePagination() {
-			const $pagination = this.$('.featured-content-pagination');
+			const pagination = this.element.querySelector('.featured-content-pagination');
 
-			$pagination.find('.current').removeClass('current');
-			$pagination.find(`li[data-index=${this.get('currentItemIndex')}]`).addClass('current');
+			pagination.querySelector('.current').classList.remove('current');
```
Comment (latata) [inline app/components/featured-content.js:121]: won't it cause any null pointer exception?

File: app/components/wikia-in-your-lang.js [hunk 5]
```diff
@@ -57,12 +57,14 @@ export default Component.extend(
...
 				callbacks: {
 					onInsertElement: (alert) => {
-						alert.on('click', 'a:not(.close)', () => {
-							track({
-								action: trackActions.click,
-								category: 'wikiaInYourLangAlert',
-								label: 'link'
-							});
+						alert.addEventListener('click', ({target}) => {
+							if (target.matches('a:not(.close)')) {
```
Comment (latata) [inline app/components/wikia-in-your-lang.js]: use `closest` here

File: app/components/category-members-grouped.js (hunk 3 above)
Comment (hakubo) [reply app/components/category-members-grouped.js:38]: I'd say not for now, if browser does not support it, it'll just scroll immediately

**Review by bkoval — APPROVED** [review]

**Pull request merged**